src/app/(authenticated)/stickers/components/create-sticker-form.tsx

```
@@ -99,170 +106,295 @@ export function CreateStickerForm({ onStickerCreated, trigger }: CreateStickerFo
+            setAvailableStock([]);
+        }
+    }, [form.watch("stickerTypeId")]);
+
+    const onSubmit = async (values: FormValues) => {
         try {
             setIsLoading(true);
-            const response = await fetch("/api/stickers", {
+
+            // Remove empty policyId if it's not provided
+            if (!values.policyId) {
+                delete values.policyId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)